aws/vpc/main.tf

```diff
@@ -1,10 +1,10 @@
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "3.11.0"
+  version = "5.1.0"
 
   name                 = "vpc-${var.cluster_name}"
   cidr                 = var.cidr
-  azs                  = data.aws_ec2_instance_type_offerings.supported_azs.locations
+  azs                  = sort(data.aws_ec2_instance_type_offerings.supported_azs.locations) #  might return different array someday, not sure how to "pin" it once it's created
```

Comment:
this caused VPC module to be recreated, didn't find a way to make it "immutable", because maybe one day an AZ will be added and break the state.